docs/roadmap.md

```diff
@@ -0,0 +1,24 @@
+# Roadmap
+
+## Version 1.0.0
+
+There are a few PRs against Webpack to use directly or indirectly this project.
+
+It's important that the features used in Webpack are working, and thus the following work is priority:
+- Making sure that we can decode WASM binaries
+  - [ ] Ability to import memory (https://github.com/xtuc/js-webassembly-interpreter/issues/137)
+  - [ ] Decode data sections
+  - [ ] Decode custom sections
+  - [ ] Add more invalid modules coverage from spec tests
+  - [x] Decode the start section (https://github.com/xtuc/js-webassembly-interpreter/pull/164)
+    - [ ] WASM - Start the execution from the start function
+    - [x] WAST - Start the execution from the start function
+    
+## Version 1.1.0
+
+The second important feature for Webpack is [wasm-dce](https://github.com/xtuc/wasm-dce).
+
+It depends on webassembly-interpreter, and thus the following work is todo:
+- Improve WASM support
+  - [ ] WASM code generation (https://github.com/xtuc/js-webassembly-interpreter/issues/68)
+  - [ ] Transform our current WAST test suite to WASM and relaunch the same tests.
```

Comment:
I think the overall test approach could do with being documented. There are extensive tests for:

1. The WAT parser
2. The interpreter
3. And a growing set of tests based on the spec test suite

The testing around WASM is less thorough, and could potentially make use of all of the above.

Reply:
I can reword it like that: 
> use our existing WAT and interpreter tests against our WASM implementation.